gst/nnstreamer/nnstreamer_conf.c

```diff
@@ -55,6 +55,7 @@ typedef struct
   gchar *pathDECODERS[CONF_SOURCES];       /**< directory paths for DECODERS */
   gchar *pathCUSTOM_FILTERS[CONF_SOURCES]; /**< directory paths for CUSTOM FILTERS */
   gchar *valueTF_MEM_OPTMZ[CONF_SOURCES];  /**< value of TF_MEM_OPTMZ */
+  gchar *valueTORCH_USE_GPU[CONF_SOURCES]; /**< value of TORCH_USE_GPU */
```

Comment:
Plz consider to use custom-bool function, not a default conf str.
(See PR #1433 for tensorflow changes)

Reply:
Updated.